src/sideEffect/saga/auth.js

```diff
@@ -22,7 +22,7 @@ export default (authClient) => {
             try {
                 yield put({ type: USER_LOGIN_LOADING });
                 yield call(authClient, AUTH_LOGIN, payload);
-                yield put({ type: USER_LOGIN_SUCCESS });
+                yield put({ type: USER_LOGIN_SUCCESS, payload });
```

Comment:
I think you'd better pass the result of the call from line 24 as the payload. That way, you grab whatever the `authClient` decides to expose (login and password, but also why not: tokens, roles, preferences, etc).